source/_posts/2015-03-26-creating-a-dynamic-single-page-app-for-our-genome-team-using-react.markdown

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title: "Trying out React"
+date: 2015-03-26 10:42
+comments: true
+categories: [React, JavaScript, CoffeeScript]
+author: Sarah Weir
+github-url: https://www.github.com/sweir27
+twitter-url: http://twitter.com/sweir27
+---
+We recently picked up a Rails application that was a few features away from completion. This application allows our [Genome Team](https://www.artsy.net/about/the-art-genome-project) to classify multiple artworks based on visual and art historical characteristics. These characteristics, or "genes", can be added, removed, and changed for any of the artworks on the panel. 
+
+![screenshot](http://cl.ly/image/310O3U3J3T2c)
+
+Our genomers are masters of efficiency, and over the years they have worked closely with our engineering team to tailor a dynamic interface to their specific workflow. 
+
+When we started working on the app, the back-end was organized, modular, and interfaced seamlessly with the Artsy API, but there were still a few front-end features we needed to implement before it could be used by the Genome Team. The app did not use a front-end framework, and as our features scaled it was difficult to keep track of UI state with pure CoffeeScript and global event listeners. Eventually, we decided to stop trying to patch our leaky roof and build a new one.
+
+### Choosing a Suitable Framework 
+
+In order to make it easier to add new features, we decided to introduce a front-end framework. We spent a day researching different frameworks to see what would best fit our needs. Our requirements were:
```

Comment:
different frameworks => different options (just to avoid saying framework twice)